templatetags/htk_tags.py

```diff
@@ -219,6 +219,26 @@ def get_request_duration():
     duration = timer.duration()
     return duration
 
+@register.simple_tag(takes_context=True)
+def has_permission(context, codename):
+    user = context.get('user', None)
+    # UserModel = get_user_model()
+    # user = UserModel.objects.get(id=user_id)
```

Comment:
from the docs you sent me:

https://docs.djangoproject.com/en/1.11/topics/auth/default/#default-permissions

You just need to do `user.has_perm(codename)`, I believe